src/sage/schemes/plane_conics/con_rational_function_field.py

```diff
@@ -130,19 +130,16 @@ def has_rational_point(self, point=False, algorithm='default',
             sage: C = Conic(K, [t^2 - 2, 2*t^3, -2*t^3 - 13*t^2 - 2*t + 18])
             sage: C.has_rational_point(point=True)
             (True, (-3 : (t + 1)/t : 1))
-
-            sage: R.<t> = FiniteField(23)[]
-            sage: C = Conic([2, t^2 + 1, t^2 + 5])
-            sage: C.has_rational_point()
+            sage: R.<t> = FiniteField(23)[]                                             # optional - sage.rings.finite_rings
+            sage: C = Conic([2, t^2 + 1, t^2 + 5])                                      # optional - sage.rings.finite_rings
+            sage: C.has_rational_point()                                                # optional - sage.rings.finite_rings
             True
-            sage: C.has_rational_point(point=True)
+            sage: C.has_rational_point(point=True)                                      # optional - sage.rings.finite_rings
             (True, (5*t : 8 : 1))
-
-            sage: # needs sage.rings.number_field
-            sage: F.<i> = QuadraticField(-1)
-            sage: R.<t> = F[]
```

Comment:
the version with the block `# needs` is the correct one

Reply:
@mezzarobba also coming from at #35848